src/InputWithLabel.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useRef } from "react";
+
+function InputWithLabel ({id, value, inputChange, children}) {
+    const inputRef = useRef(null);
+
+    useEffect(() => {
+        inputRef.current.focus();
+    }, []);
+
+    return (
+        <>
+            <label htmlFor={id}>Title:{children}</label>
+                <br />
+                <input
+                type="text"
+                id={id}
+                name={id}
+                required
+                value={value}
+                onChange={inputChange}
+                ref = {inputRef}
+            />
```

Comment:
The inner workings of the component should have just been copy/pasted from `AddTodoForm`, and I'm not sure where you were getting these other props from like `inputChange`. Below is what this component should look like

```
import React, { useEffect, useRef } from "react";

const InputWithLabel = ({ children, todoTitle, handleTitleChange }) => {
  const inputRef = useRef();

  useEffect(() => {
    inputRef.current.focus();
  });

  return (
    <>
      <label htmlFor="inputField">{children}</label>
      <input
        type="text"
        id="inputField"
        name="title"
        value={todoTitle}
        onChange={handleTitleChange}
        required
        ref={inputRef}
      />
    </>
  );
};

export default InputWithLabel;

```